src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx

```diff
@@ -38,19 +38,23 @@ const AppsCarousel: React.FC<Props> = (props: Props) => {
       {items.map((item, index) => {
         const { name, icon, description, tag, ctaLink } = item;
         return view === index ? (
-          <Box className={clsx(classes.cardContainer, "expandCard", { open: inView })} style={{ backgroundImage: `linear-gradient(52.83deg, #121212 15.94%, rgba(0, 0, 0, 0) 100%), url("${item?.backgroundImage}")`, boxShadow: `${theme.shadows[5]}`, backgroundSize: widthSm ? "200%" : "" }} key={`${name}-featured-dApps`}>
-            <Typography variant="body1" color="textPrimary" className={classes.tag}>{tag}</Typography>
-            <img src={icon} className={classes.dAppLogo} />
-            <Typography variant="h3" color="textPrimary" style={{ fontFamily: "TyrosPro-Bold", fontWeight: 700, marginTop: "1rem" }}>{name}</Typography>
-            <Typography variant="body1" style={{ color: theme.palette.text.button, marginTop: "1rem" }}>{description}</Typography>
-            <Button className={classes.ctaButton} href={ctaLink} target="_blank">Launch {name}</Button>
-          </Box >
+          <Grow in timeout={(index + 1) * 200 > 1000 ? 1000 : (index + 1) * 200} key={`${name}-featured-dApps`}>
+            <Box className={clsx(classes.cardContainer, "expandCard", { open: inView })} style={{ backgroundImage: `linear-gradient(52.83deg, #121212 15.94%, rgba(0, 0, 0, 0) 100%), url("${item?.backgroundImage}")`, boxShadow: `${theme.shadows[5]}`, backgroundSize: widthSm ? "200%" : "" }}>
+              <Typography variant="body1" color="textPrimary" className={classes.tag}>{tag}</Typography>
+              <img src={icon} className={classes.dAppLogo} />
+              <Typography variant="h3" color="textPrimary" style={{ fontFamily: "TyrosPro-Bold", fontWeight: 700, marginTop: "1rem" }}>{name}</Typography>
+              <Typography variant="body1" style={{ color: theme.palette.text.button, marginTop: "1rem" }}>{description}</Typography>
+              <Button className={classes.ctaButton} href={ctaLink} target="_blank">Launch {name}</Button>
+            </Box >
+          </Grow>
         ) : (
-          <Box className={clsx(classes.cardContainer, { open: inView })} key={`${name}-featured-dApps`}>
-            <img src={icon} className={classes.logo} />
-            <Typography variant="h4" color="textPrimary" style={{ fontFamily: "TyrosPro-Bold", fontWeight: 700, marginTop: "1rem" }}>{name}</Typography>
-            <Button className={clsx(classes.ctaButton, classes.minButton)} href={ctaLink} target="_blank">{tag}</Button>
-          </Box>
+          <Grow in timeout={(index + 1) * 200 > 1000 ? 1000 : (index + 1) * 200} key={`${name}-featured-dApps`}>
+            <Box className={clsx(classes.cardContainer, { open: inView })}>
+              <img src={icon} className={classes.logo} />
+              <Typography variant="h4" color="textPrimary" style={{ fontFamily: "TyrosPro-Bold", fontWeight: 700, marginTop: "1rem" }}>{name}</Typography>
```

Comment:
The same styles is used elsewhere, move the styles into a class and apply it here.